src/components/booksComponent/BooksItem.js

```diff
@@ -1,11 +1,29 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const BooksItem = () => (
-  <div>
-    <h2>Mein Kampf</h2>
-    <p>Adolf Hitler</p>
-    <button type="button">Delete</button>
-  </div>
-);
+const BooksItem = (props) => {
+  const {
+    id, title, author, propsToDeleteBook,
+  } = props;
+
+  const deleteBook = (e) => {
+    const bookId = e.target.dataset.bookid;
+    propsToDeleteBook(bookId);
+  };
+  return (
+    <div>
+      <h2>{title}</h2>
+      <p>{author}</p>
+      <button onClick={deleteBook} data-bookid={id} type="button">Delete</button>
+    </div>
+  );
+};
+
+BooksItem.propTypes = {
+  id: PropTypes.number.isRequired,
```

Comment:
- [ ] This PropType definition presents us with the error below. As you can see from the screenshot, the id is actually a string when a book is created. Kindly work on this.

  ![image](https://user-images.githubusercontent.com/45002654/149952526-dfe4b833-7885-4263-887c-44b039583752.png)